src/runtime/runtime.ts

```diff
@@ -56,37 +60,37 @@ export type RuntimeArcOptions = Readonly<{
   modality?: Modality;
 }>;
 
+
+// TODO(sjmiles): weird layering here due to dancing around global state (working on it)
+let staticMemoryProvider;
+const initDrivers = () => {
+  VolatileStorageKey.register();
```

Comment:
this is a change in behavior, right? originally one would have to register ramdisk manually.
also, isn't it weird that VolatileStrorageKey is being registered, but ramdisk storage provider?